design/decisions/why-containers/Why_Container_Document.md

```diff
@@ -0,0 +1,34 @@
+# Reason for Choosing Containerization Technology
+
+## What is Containerization
+
+Containerization is the technology of packaging software development code along with all of its necessary dependencies or configurations, including as libraries, frameworks, and other dependencies, into a single, isolated "container." The software or application inside the "container" can be transported, deployed and used reliably in any environment and on any infrastructure, regardless of the operating system. The container functions as a sort of computing environment or bubble around the software or application, keeping it apart from the actual computing environment. It made moving or deploying the software or application to any new computing environment less complicated.
+
+## Benefit of Containerization
```

Comment:
Do we also need a section where we look at the drawbacks compared with running a traditional server set-up.

Reply:
Sure, I could add a section for "drawbacks of the Containerization"